.github/workflows/REUSE.wheelbuild.yaml

```diff
@@ -0,0 +1,53 @@
+name: Wheel builder
+
```

Comment:
What does the `REUSE` prefix signify here, that this is a reusable component ?

Reply:
Reading more I understand now the answer is yes, this is used in both the manual wheelbuilder and the publish to pypi, but with different inputs.

Is inputs being a place in the yaml for arguments a common github actions convention?

Reply:
Correct. By prefixing the job REUSE we signal to other users that we're creating a reusable component. One could place this outside of the workflows directory, given the pathing, but my intuition is such that someone looks for GitHub workflows in workflows, so we might as well pack as much as possible within this space.